packages/storybook-vue/stories/components/beta-brand-header-navigation/BrandHeader.stories.mdx

```diff
@@ -178,6 +178,34 @@ scale-telekom-nav-list {
 </scale-telekom-header>
 ```
 
+## Dropdown Menu
```

Comment:
@felix-ico also I think this one should be called `Dropdown Navigation`, no?

Reply:
hmm not sure about this one, we had a few issues with the naming but did not receive any feedback on this one specifically..

Reply:
ok I got a change requests from Joscha too about this, updated!